src/components/Dependencies/Dependencies.test.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { HashRouter } from 'react-router-dom';
+
+import { DetailContextProvider } from '../../pages/Version';
+
+import Dependencies from './Dependencies';
+
+describe('<Dependencies /> component', () => {
+  test('Renders a message when there are no dependencies', () => {
+    // Given
+    const packageMeta = {
+      latest: {
+        name: 'verdaccio',
+        version: '4.0.0',
+        author: {
+          name: 'verdaccio user',
+          email: 'verdaccio.user@verdaccio.org',
+          url: '',
+          avatar: 'https://www.gravatar.com/avatar/000000',
+        },
+        dist: { fileCount: 0, unpackedSize: 0 },
+        dependencies: {},
+        devDependencies: {},
+        peerDependencies: {},
+      },
+      _uplinks: {},
+    };
+
+    // When
+    const { getByText } = render(
+      <DetailContextProvider value={{ packageMeta }}>
+        <Dependencies />
+      </DetailContextProvider>
+    );
+
+    // Then
+    expect(getByText('verdaccio has no dependencies.')).toBeDefined();
+  });
+
+  test('Renders a link to each dependency', () => {
+    // Given
+    const packageMeta = {
```

Comment:
Maybe you could make the packageMeta reusable?

Reply:
I agree ! We should start implementing reusable fixtures for the whole project. My next PR ;)

Reply:
Perhaps this helps https://github.com/verdaccio/verdaccio/blob/master/test/unit/__helper/utils.ts

Reply:
How about we use this in a separate PR ?